<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_data</class>
 <widget class="QDialog" name="Edit_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Данные</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/img/pic/Autorization/auto_icon.ico</normaloff>:/img/pic/Autorization/auto_icon.ico</iconset>
  </property>
  <widget class="QLabel" name="login">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monotype Corsiva</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Логин:</string>
   </property>
  </widget>
  <widget class="QLabel" name="old_pass">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monotype Corsiva</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Старый пароль:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="login_input">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;
border: 1px solid grey;
color: black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="old_passw_input">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;
border: 1px solid grey;
color: black;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_pass_input">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;
border: 1px solid grey;
color: black;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirm_pass_input">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 8px;
border: 1px solid grey;
color: black;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="new_pass">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monotype Corsiva</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Новый пароль:</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirm_pass">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monotype Corsiva</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Подтверждение:</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>240</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/img/pic/MainWindow/bg.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="done">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>190</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-style: outset;
     min-width: 5em;
     padding: 6px;
     border: 1px solid #404040;
     /*border-width: 2px;*/
     border-radius: 8px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #999999, stop: 1 #404040);
	color: white;
}

QPushButton:pressed {
	 color:rgb(200,200,200);
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #404040, stop: 1 #999999);
 }

QPushButton:disabled {
	color:rgb(200,200,200);
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
									  stop: 0 #555555, stop: 1 #999999);
}

 QPushButton:hover {
	 border: 1px solid #404040;
     border-width: 2px;
     border-color: green;
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>110</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border-style: outset;
     min-width: 5em;
     padding: 6px;
     border: 1px solid #404040;
     /*border-width: 2px;*/
     border-radius: 8px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #999999, stop: 1 #404040);
	color: white;
}

QPushButton:pressed {
	 color:rgb(200,200,200);
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #404040, stop: 1 #999999);
 }

QPushButton:disabled {
	color:rgb(200,200,200);
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
									  stop: 0 #555555, stop: 1 #999999);
}

 QPushButton:hover {
	 border: 1px solid #404040;
     border-width: 2px;
     border-color: green;
 }

 QPushButton:default {
     border-color: navy; /* делаем кнопку по умолчанию выпуклой */
 }</string>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>done</zorder>
  <zorder>login</zorder>
  <zorder>old_pass</zorder>
  <zorder>cancel</zorder>
  <zorder>login_input</zorder>
  <zorder>old_passw_input</zorder>
  <zorder>new_pass_input</zorder>
  <zorder>confirm_pass_input</zorder>
  <zorder>new_pass</zorder>
  <zorder>confirm_pass</zorder>
  <zorder>done</zorder>
  <zorder>cancel</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
